transpire/helpers/helm.py

```diff
@@ -95,6 +95,35 @@ def build_chart(
     if shutil.which("helm") is None:
         raise RuntimeError("You must install Helm to use this script.")
 
+    # make sure the current user has a place for us to cache stuff
+    home_cache = os.environ.get('XDG_CACHE_HOME', None)
+    if not home_cache:
+        raise EnvironmentError("Could not find an XDG_CACHE_HOME environment variable")
```

Comment:
We definitely don't want to fail if `$XDG_CACHE_HOME` isn't set. In this case we probably want to default to `$HOME/.cache/transpire`.